src/icp_canister_backend/tests/rewards_tests.rs

```diff
@@ -1,10 +1,11 @@
 use candid::{decode_one, encode_args, Nat, Principal};
 use icp_canister_backend::EPISODE_DURATION;
-
 mod setup;
 use setup::setup;
 mod utils;
-use utils::{advance_time, create_deposit, get_stakable_episode, reward_pool, ALLOWED_ERROR};
+use utils::{
+    advance_time, create_deposit, get_stakable_episode, reward_pool, ALLOWED_ERROR
+};
 
 #[test]
 fn test_reward_rate_increase_decrease_during_episodes() {
```

Comment:
Also add to this test, that if you increase time even, a user deposit doesn't receive additional rewards, after the expected reward rate drop.